packages/@aws-cdk/cli-lib-alpha/lib/cli.ts

```diff
@@ -1,7 +1,7 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
-import { exec as runCli } from 'aws-cdk/lib';
+import { exec as runCli } from '../../../aws-cdk/lib';
```

Comment:
Surely, not this?

Reply:
This is intentional. We don't want to depended on `aws-cdk` the package, but on the local file. It works because `cli-lib-alpha` is bundling everything itself.

Reply:
This doesn't make any practical difference. For me it's about conveying an intention. But happy to revert if you feel strongly otherwise.